packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -54,411 +53,50 @@ export const Default = {
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'expandable' }
-      },
-      args,
-      constants: { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' }
-    });
-  }
-};
-
-/**
- * This shows sd-quickfact in a mobile view. **Please navigate to the `Mobile` story** (you are now on the `Docs` page) to accurately view this behavior.
-
- */
-
-export const Mobile = {
-  render: (args: any) => {
-    return generateTemplate({
-      args,
-      constants: { type: 'attribute', name: 'expandable', value: 'true' }
-    });
-  },
-  parameters: {
-    viewport: {
-      viewports: INITIAL_VIEWPORTS,
-      defaultViewport: 'iphone6'
-    }
-  }
+  render: () => html`
+    <sd-quickfact expandable="" summary="Lorem Ipsum">
+      <sd-icon name="content/image" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+      <div class="slot slot--border slot--text h-16">Default slot</div>
+    </sd-quickfact>
+  `
 };
 
 /**
- * The summary can have multiple lines.
+ * Use the `open` attribute to set the state of the quickfact.
  */
-export const SummaryLength = {
-  parameters: { controls: { exclude: 'summary' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'slot',
-          name: 'summary',
-          values: [
-            { value: '<slot slot="summary">Lorem ipsum.</slot>', title: 'short' },
-            {
-              value:
-                '<slot slot="summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</slot>',
-              title: 'long'
-            }
-          ]
-        }
-      },
-      args,
-      constants: [
-        { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
-        { type: 'slot', name: 'summary', value: '<div slot="summary" class="slot slot--text">Test</div>`' }
-      ]
-    });
-  }
+export const Open = {
+  render: () => html`
+    <div class="flex flex-col gap-12">
+      <sd-quickfact expandable="" open summary="Open">
+        <sd-icon name="content/image" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+        <div class="slot slot--border slot--text h-16">Default slot</div>
+      </sd-quickfact>
+
+      <sd-quickfact expandable="" summary="Closed">
+        <sd-icon name="content/image" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+        <div class="slot slot--border slot--text h-16">Default slot</div>
+      </sd-quickfact>
+    </div>
+  `
 };
 
 /**
- * Use the expand-icon and collapse-icon slots to change the expand and collapse icons, respectively.
- * To disable the animation, override the rotate property on the summary-icon part as shown below:
+ * Use the `summary` attribute to provide the text in the accordion quickfact.
```

Comment:
* Use the `summary` attribute to provide the text in the quickfact.

Reply:
Updated